README.md

```diff
@@ -1,16 +1,166 @@
-![LUSID_by_Finbourne](https://content.finbourne.com/LUSID_repo.png)
+# LUSID<sup>®</sup> Python SDK
+This is the Python SDK for [LUSID by FINBOURNE](https://www.finbourne.com/lusid-technology), a bi-temporal investment management data platform with portfolio accounting capabilities. To use it you'll need a LUSID account. [Sign up for free at lusid.com](https://www.lusid.com/app/signup)
 
-| branch | status |
-| --- | --- |
-| `master` | ![PyPI](https://img.shields.io/pypi/v/lusid-sdk?color=blue) ![run-sdk-tests](https://github.com/finbourne/lusid-sdk-python/workflows/run-sdk-tests/badge.svg?branch=master) [![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=finbourne_lusid-sdk-python&metric=alert_status)](https://sonarcloud.io/dashboard?id=finbourne_lusid-sdk-python) |
-| `develop` | ![run-sdk-tests](https://github.com/finbourne/lusid-sdk-python/workflows/run-sdk-tests/badge.svg?branch=develop) |
+<a href="https://www.lusid.com/app/signup"><img src="https://content.finbourne.com/LUSID_repo.png" alt="LUSID_by_Finbourne" width="250"></a>
```

Comment:
did you mean to resize the banner? I think the full width one looks better

Reply:
Yeh - I was playing around with the size a bit, but mostly I wanted to move the title + description of the repo to the top, so we can put the size back how it was